e2e/playwright/st_toggle.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import streamlit as st
+from streamlit import runtime
+
```

Comment:
Nit: Prefer that the widget label describes the case tested for clarity, something like: 
`toggle 1 - value True`
`toggle 2 - value False` ...

Reply:
Sounds good 👍 I changed it for toggle and checkbox e2e tests

Reply:
😅 Sorry for confusion, was referring to the `st.toggle` widget labels since those appear in the screenshots and makes it more explicit what each snapshot should show (not the `st.write` text)

Reply:
Oh, sorry I misinterpreted the comment 😅 I updated that as well 👍